cli.js

```
@@ -49,21 +49,19 @@ function action(paths, options) {
   }
 
   if (files.length === 0) {
-    console.error('No files were found.');
-    process.exit(1);
+    throw new Error('No files were found.');
   }
 
   files = files.filter(function(element, index, array) {
     return (['pdf', 'jpeg', 'jpg', 'gif', 'png', 'bmp'].indexOf(element.split('.').pop().toLowerCase()) !== -1);
   });
 
   if (files.length === 0) {
-    console.error('No image or PDF files could be found.');
-    process.exit(1);
+    throw new Error('No image or PDF files could be found.');
```

We should throw an error instead of console.error